app/app/src/main/java/org/omecproject/up4/cli/FlowsStatsCommand.java

```diff
@@ -0,0 +1,108 @@
+/*
+ SPDX-License-Identifier: Apache-2.0
+ SPDX-FileCopyrightText: 2022-present Intel Corporation
+ */
+package org.omecproject.up4.cli;
+
+import com.google.common.collect.Maps;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.omecproject.up4.impl.Up4AdminService;
+import org.onlab.packet.Ip4Address;
+import org.onosproject.cli.AbstractShellCommand;
+import org.onosproject.net.behaviour.upf.UpfCounter;
+
+import java.util.Map;
+
+/**
+ * UP4 UE flow stats.
+ */
+@Service
+@Command(scope = "up4", name = "flow-stats",
+        description = "Read all UE flows stats")
+public class FlowsStatsCommand extends AbstractShellCommand {
```

Comment:
I'm thinking about the case where we want to troubleshoot a specific UE. In that case I would like to monitor stats for the specific UE only and I want to do it over time. I.e., leave the stats command running while a start iperf or similar. Can we add a parameter to filter stats for a specific UE? Does it make sense to continue reading until we kill the command? Or maybe assuming someone will us `watch flow-stats` is enough?

Reply:
Added the possibility of filtering by UE Address. About doing it over time, I would prefer using `watch`, so we can reuse the command for multiple use.

Reply:
What happens if the watch interval > sleep time?

Reply:
Changes approach. Added the `-c` option, to continuously read and print stats.